sql/item_mods.sql

```diff
@@ -36743,7 +36768,7 @@ INSERT INTO `item_mods` VALUES (18329,356,185); -- ADDS_WEAPONSKILL_DYN: 185
 INSERT INTO `item_mods` VALUES (18330,25,20);   -- ACC: 20
 INSERT INTO `item_mods` VALUES (18330,256,12);  -- AFTERMATH: 12
 INSERT INTO `item_mods` VALUES (18330,355,185); -- ADDS_WEAPONSKILL: 185
-INSERT INTO `item_mods` VALUES (18330,431,10);   -- ITEM_ADDEFFECT_TYPE: 10
+INSERT INTO `item_mods` VALUES (18330,431,10);   -- ITEM_ADDEFFECT_TYPE: DAMAGE0
```

Comment:
Theres a few cases of this, where `10` has been replaced with `DAMAGE0`
Shouldnt this be `DISPEL`?

```lua
    local procType =
    {
        -- These are arbitrary, make up new ones as needed.
        DAMAGE        = 1,
        DEBUFF        = 2,
        HP_HEAL       = 3,
        MP_HEAL       = 4,
        HP_DRAIN      = 5,
        MP_DRAIN      = 6,
        TP_DRAIN      = 7,
        HPMP_DRAIN    = 8,
        HPMPTP_DRAIN  = 9,
        DISPEL        = 10,
        ABSORB_STATUS = 11,
        SELF_BUFF     = 12,
        DEATH         = 13,
    }
```

Reply:
find+replace issue, should be fixed now but github is refusing to acknowledge it on this comment thread (when I go to the actual file her eon github, I do see my pushed change though)